corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -180,7 +181,7 @@ def inline_edit_trans(name, langs=None, url='', saveValueName='', postSave='',
     return format_html(template, **options)
 
 
-def _get_dynamic_input_trans_options(name, langs=None, allow_blank=True):
+def _get_dynamic_input_trans_options(name, langs=None, allow_blank=True, value_escape="js"):
```

Comment:
Idle nitpick: I wonder if it'd be clearer to name `value_escape` something like `is_textarea`. I read through the usage of `value_escape` a couple of times to be sure I understood, because seeing "escape" catches my eye and makes me worry about security issues. I think it's easier for a user of this function to answer the question "What value should I pass to `is_textarea`?" than to answer "What value should I pass to `value_escape`?" because it may not be top of mind for them that values are attributes for input and in-element text for textareas.

This is largely academic because this function is only used in this file and not going to have many callers, just food for thought.

Reply:
Makes sense. I thought for a while what would make sense - I was attempting to keep the variable name more specific in describing its purpose but I agree `value_escape` is confusing.

Changed to `is_textarea` in commit [9283977](https://github.com/dimagi/commcare-hq/pull/32491/commits/9283977a0d86bdf16a1640c7e42c5bd7a60b2117)